### PR TITLE
(Resubmission) Spinning Wheel Workshop

### PR DESCRIPTION
**Pull request by HariOm987**
@MatthewStanciu  I have completed the workshop and checked its grammar, punctuation and phrasing. I guess this time it will get merged. :) 
[Meeting Recording](https://hackclub.slack.com/files/U019BA048CT/F01E7AH8K3K/workshop__recording.mp4)
🤞🤞🤞

### CODE REVIEW

File: workshops/spinning-wheel/README.md [hunk 1]
```diff
@@ -0,0 +1,464 @@
+---
+name: 'Spin The Wheel'
+description: 'Build a random picker spinning wheel using HTML, CSS, and JavaScript'
+author: '@HariOm987'
+img: https://cloud-gvkfrv1qw.vercel.app/0image.png
+---
+
+In this workshop, you're going to make a colorful spinning wheel with your own custom content, in 20 minutes!
+
+Here are [live demo](https://movie-selector.hariom04.repl.co/) and the [final code](https://repl.it/@hariom04/movie-selector#index.html).
```
Comment (sampoder) [inline workshops/spinning-wheel/README.md:10]: check the grammar / phrasing here

File: workshops/spinning-wheel/README.md [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+
+![Final Output of our Workshop](https://cloud-4945tg5q1.vercel.app/0final.png)
+
+^^^ If you will follow this workshop strictly then you will make something like the above picture, but feel free to add up your imaginations with patterns and colors and you will end up with something better than this. So let's begin. 😁✨
```
Comment (sampoder) [inline workshops/spinning-wheel/README.md:14]: here too

File: workshops/spinning-wheel/README.md [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+
+# Part 1: Prerequisites
+* Account on Repl.it
+* Basic knowledge of HTML, CSS and JavaScript would be helpful but is not necessary to continue further.
+
+# Part 2: Setup
+## Setting up our code environment
+In this workshop, we'll using repl.it, a free, online code editor.
```
Comment (sampoder) [inline workshops/spinning-wheel/README.md:22]: here three?

**Comment by MatthewStanciu** [comment]
Yaaaaay 🎉

**Comment by HariOm987** [comment]
🥳🥳🥳Thank you so much😍

**Pull request merged**